src/vs-bicep/Bicep.VSLanguageServerClient.Vsix/packages.lock.json

```diff
@@ -1829,6 +1831,12 @@
           "Microsoft.Visualstudio.Telemetry": "[17.11.20, )",
           "OmniSharp.Extensions.LanguageServer": "[0.19.9, )"
         }
+      },
+      "bicep.vslanguageserverclient.itemtemplate": {
+        "type": "Project",
+        "dependencies": {
+          "Microsoft.VisualStudio.CoreUtility": "[17.11.260, )"
+        }
```

Comment:
I'm not sure what entries like this in a lockfile mean. This was added automatically by VS.

Reply:
It seems to be added and removed sporadically.  Not sure if it matters whether it's in or not.